TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithCommit.rdz.cbl

```diff
@@ -6,7 +6,12 @@
          COMMIT
          END-EXEC.
        PROCEDURE DIVISION.
-           EXEC SQL COMMIT END-EXEC display "hello" EXEC SQL COMMIT
+           EXEC SQL COMMIT 
```

Comment:
No. We need to keep the original test line. The idea is to test unusual cases like this one where two EXEC SQL are on the same line.

Just add your new test cases after the existing one, so at line 11 you can add your new `EXEC SQL COMMIT WORK`.